src/js/jquery.slides.coffee

```diff
@@ -0,0 +1,796 @@
+# SlidesJS 3.0.4
```

Comment:
You can leave this file out if you want, since it's the original CoffeeScript file